### ISSUE TITLE
Luma Home page creating google pagespeed accessibility errors

### ISSUE THREAD

**Issue by fabotha** [magento/magento2#40200]
### Preconditions and environment

- Magento version 2.4.8 Using Luma Theme
- Anything else that would help a developer reproduce the bug
- web page getting error www.firesupport.uk

### Steps to reproduce

Create categories on the web site that display on the home page

### Expected result

Pass google accessibility checks

### Actual result

Fails google page speed accessibility tests

### Additional information

Google Page Speed errors
Elements with an ARIA [role] that require children to contain a specific [role] are missing some or all of those required children.
Some ARIA parent roles must contain specific child roles to perform their intended accessibility functions. Learn more about roles and required children elements.
Failing elements
Airsoft GunsBBs Batteries GasMagazinesChrono Tools ChargersReplacement PartsSco…
`<div class="section-items nav-sections-items mage-tabs-disabled" role="tablist">`
Airsoft GunsBBs Batteries GasMagazinesChrono Tools ChargersReplacement PartsSco…
`<div class="section-item-content nav-sections-item-content" id="store.menu" data-role="content" role="tabpanel" aria-hidden="false">` Airsoft GunsBBs Batteries GasMagazinesChrono Tools ChargersReplacement PartsSco…
`<ul id="ui-id-1" role="menu" tabindex="0" class="ui-menu ui-widget ui-widget-content">`
Airsoft Guns
`<li class="level0 nav-1 category-item first level-top ui-menu-item">`
BBs Batteries Gas
`<li class="level0 nav-2 category-item level-top ui-menu-item">`
Magazines
`<li class="level0 nav-3 category-item level-top ui-menu-item">`

also

[role]s are not contained by their required parent element
Some ARIA child roles must be contained by specific parent roles to properly perform their intended accessibility functions. Learn more about ARIA roles and required parent element.
Failing elements
Airsoft Guns
`<a href="https://www.firesupport.uk/airsoft-guns.html" class="level-top ui-menu-item-wrapper" id="ui-id-2" tabindex="-1" role="menuitem">`
BBs Batteries Gas
`<a href="https://www.firesupport.uk/consumeables.html" class="level-top ui-menu-item-wrapper" id="ui-id-3" tabindex="-1" role="menuitem">`
Magazines
`<a href="https://www.firesupport.uk/magazines.html" class="level-top ui-menu-item-wrapper" id="ui-id-4" tabindex="-1" role="menuitem">`
Chrono Tools Chargers
`<a href="https://www.firesupport.uk/maintenance.html" class="level-top ui-menu-item-wrapper" id="ui-id-5" tabindex="-1" role="menuitem">`
Replacement Parts
`<a href="https://www.firesupport.uk/replacement-parts.html" class="level-top ui-menu-item-wrapper" id="ui-id-6" tabindex="-1" role="menuitem">`
Scope Grip Rail Laser
`<a href="https://www.firesupport.uk/accessories.html" class="level-top ui-menu-item-wrapper" id="ui-id-7" tabindex="-1" role="menuitem">`
More Item
`<a href="https://www.firesupport.uk/more-items.html" class="level-top ui-menu-item-wrapper" aria-haspopup="true" id="ui-id-8" tabindex="-1" role="menuitem">`
New Product
`<a href="https://www.firesupport.uk/new-products.html" class="level-top ui-menu-item-wrapper" aria-haspopup="true" id="ui-id-12" tabindex="-1" role="menuitem">`
Sale
`<a href="https://www.firesupport.uk/sale-items.html" class="level-top ui-menu-item-wrapper" aria-haspopup="true" id="ui-id-69" tabindex="-1" role="menuitem">`

![Image](https://github.com/user-attachments/assets/5fa39a92-950a-4d2e-b18e-d23d33003ca2)
![Image](https://github.com/user-attachments/assets/31ce8c77-1ef9-406b-819a-9f896acbc83a)
![Image](https://github.com/user-attachments/assets/6d3f4d71-057a-4d1a-aa19-d359e3243e6f)
![Image](https://github.com/user-attachments/assets/00fab21d-d22d-462f-a08b-fd666f53516c)

and

Tables and lists
List items `(<li>)` are not contained within `<ul>, <ol> or <menu> ` parent elements.
Screen readers require list items `(<li>)` to be contained within a parent `<ul>, <ol> or <menu>` to be announced properly. Learn more about proper list structure.

Airsoft Guns
`<li class="level0 nav-1 category-item first level-top ui-menu-item">`
BBs Batteries Gas
`<li class="level0 nav-2 category-item level-top ui-menu-item">`
Magazines
`<li class="level0 nav-3 category-item level-top ui-menu-item">`
Chrono Tools Chargers
`<li class="level0 nav-4 category-item level-top ui-menu-item">`
Replacement Parts
`<li class="level0 nav-5 category-item level-top ui-menu-item">`
Scope Grip Rail Laser
`<li class="level0 nav-6 category-item level-top ui-menu-item">`

### Triage and priority

- [ ] Severity: **S0** _- Affects critical data or functionality and leaves users without workaround._
- [ ] Severity: **S1** _- Affects critical data or functionality and forces users to employ a workaround._
- [x] Severity: **S2** _- Affects non-critical data or functionality and forces users to employ a workaround._
- [ ] Severity: **S3** _- Affects non-critical data or functionality and does not force users to employ a workaround._
- [ ] Severity: **S4** _- Affects aesthetics, professional look and feel, “quality” or “usability”._

**Comment by engcom-November** [comment]
Hello @fabotha ,

Thank you for your report and collaboration.

We attempted to reproduce the issue in our latest version 2.4 develop but were unable to do so. Please find the attached screenshot for reference.

<img width="1711" height="869" alt="Image" src="https://github.com/user-attachments/assets/06cf5a2e-3b1f-47f1-8324-c13c497732ce" />

We followed these steps and did not encounter any issues. Could you please recheck in the latest version of Magento and let us know if we are missing any steps to reproduce this issue?

Therefore, we are marking this ticket as "Issue: Needs Update".

Thank you!

**Comment by fabotha** [comment]
I use the default theme, so how do you account for my errors with the category menus in the google accessibility tests?
What could I have done any differently?
I have some extensions, could they be affecting the layout?
